docs/core-concepts/model-basics.md

```diff
@@ -278,6 +278,34 @@ Foo.init({ /* attributes */ }, {
 });
 ```
 
+## Prevent creating a default PK attribute
+
+By default, Sequelize automatically adds the primary key attribute `id` to every model when there is no primary key has been defined manually. To prevent this you can use the option `noPrimaryKey` when creating the instance and set it to true.
+
+```js
+const User = sequelize.define('User', {
+  name: DataTypes.STRING,
+}, {
+  noPrimaryKey: true,
+});
+```
+
+In case you want to disable primary keys on every models.
```

Comment:
What do you think of this rewording? I'm proposing it because I'm afraid someone may think it means all primary keys will be disabled, even the ones added manually

```suggestion
In case you want to prevent the addition of the default foreign key for every model:
```

Reply:
> 

I think it's nice!

Reply:
It's my mistake to not aware about it